zgw_consumers/models/services.py

```diff
@@ -28,6 +32,15 @@ class Service(RestAPIService):
     uuid = models.UUIDField(_("UUID"), default=uuid.uuid4)
     api_type = models.CharField(_("type"), max_length=20, choices=APITypes.choices)
     api_root = models.CharField(_("api root url"), max_length=255, unique=True)
+    api_health_check_endpoint = models.CharField(
+        _("health check endpoint"),
+        help_text=_(
+            "An optional API endpoint which will be used to check if the API is configured correctly and "
+            "is currently up or down. This field is only used for in the admin's 'health check' field."
+        ),
+        max_length=255,
+        blank=True,
+    )
```

Comment:
```suggestion
    health_check_path = models.CharField(
        _("health check"),
        help_text=_(
            "An optional API endpoint which will be used to check if the API is configured correctly and "
            "is currently up or down. This field is only used for in the admin's 'health check' field."
        ),
        max_length=255,
        blank=True,
    )
```
endpoint implies that it's a fully qualified path/URL, and you even seem to allow entire `https://example.com/...` values which can have a different domain even than the API root. I'm not sure if this is intentional?

Either way, I think it's wise to only allow relative paths (`"foo/bar/_health"`) which need to be appended to the `api_root` for a fully qualified URL. So, you need to validators on this field:

* may not be an absolute path (leading slash)
* must not be a URL (so may not fit the `scheme://[auth@]netloc/path[...]` pattern.

Ape-pie validates that the URLs passed to requests match the `base_url`, but that's an ape-pie thing, not a zgw-consumers thing.